shell/platform/fuchsia/flutter/default_session_connection.cc

```diff
@@ -178,6 +178,8 @@ DefaultSessionConnection::DefaultSessionConnection(
       on_frame_presented_callback_(std::move(on_frame_presented_callback)),
       kMaxFramesInFlight(max_frames_in_flight),
       vsync_offset_(vsync_offset) {
+  FML_DCHECK(kMaxFramesInFlight > 0);
```

Comment:
This will never fire, because all of the flutter prebuilts are built in release and by default developer builds are too.  Unfortunately we have to use CHECKs, or change how the prebuilts are made (which we can't do ATM)

Reply:
Done